sympy/physics/hydrogen.py

```diff
@@ -8,11 +8,14 @@ def R_nl(n, l, r, Z=1):
     """
     Returns the Hydrogen radial wavefunction R_{nl}.
 
-    n, l
-        quantum numbers 'n' and 'l'
-    r
+    Parameters
+    ==========
+
+    n, l :
+        Quantum numbers 'n' and 'l'.
+    r :
```

Comment:
Maybe this should say that the parameters should be expressions representing integers. Something like:
```
n, l: Expr
    Expressions representing the quantum numbers 'n' and 'l' which should be integers in the range ...
```

Reply:
@aktech I have added a similar description here, but I am not sure about the range of integers. Could you look into this?

Reply:
As per my research, the range is following
![q-20numbers](https://user-images.githubusercontent.com/5647941/100422399-892d4e80-308a-11eb-805e-29bd9635bcc5.png)

[Reference link](https://courses.lumenlearning.com/introchem/chapter/quantum-numbers/)

Also, I think `n` and `l` should be on different lines.

Reply:
Thanks! Have added them.